strawberry: 1.1.3 -> 1.2.2 (#358024)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2505.section.md

- `buildGoPackage` has been removed. Use `buildGoModule` instead. See the [Go section in the nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#sec-language-go) for details.

> - strawberry has been updated to 1.2, which drops support for VLC backend and Qt 5.
>   The `strawberry-qt5` package has been removed. Use `strawberry-qt6` instead. Options `withGstreamer` and `withVlc`
>   have been removed, GStreamer is now the default.

Reviewer: ```suggestion
- `strawberry` has been updated to 1.2, which drops support for the VLC backend and Qt 5. The `strawberry-qt5` package and `withGstreamer`/`withVlc` override options have been removed due to this. 
```
